insitu/lib/insitu/filter.hpp

```diff
@@ -159,7 +159,7 @@ class Filter : public nodelet::Nodelet
         while (exitCond.wait_for(1ms) == std::future_status::timeout)
         {
             updateFilter(apply());
-            std::this_thread::sleep_for(10ms);
+            std::this_thread::sleep_for(15666us);
```

Comment:
15666?

Reply:
that combined with the 1ms wait above limits filter update rate to 60Hz